microsoft-edge/devtools-guide-chromium/storage/sessionstorage.md

```diff
@@ -86,7 +86,7 @@ To delete a key/value pair for session storage:
 
 1. In the **Key/Value** table on the right, select a key-value pair to highlight it.
 
-1. Press `Delete`, or click the **Delete Selected** (![Delete Selected](./sessionstorage-images/delete-icon.png)) button.
+1. Press `Delete`, or click the **Delete Selected** (![Delete Selected](./sessionstorage-images/delete-icon.png)) button, or right-click on the row and select **Delete**.
```

Comment:
```suggestion
1. Press `Delete`, or click the **Delete Selected** (![Delete Selected](./sessionstorage-images/delete-icon.png)) button, or right-click the row and then select **Delete**.
```

Reply:
done locally

Reply:
i'm seeing a different version of the .md than expected.  due to forked PR??  investigating...

Reply:
in GH Desktop I had to pull 137 commits in this branch that's connected to the forked PR